src/app/screens/sign/components/sign-in/sign-in.component.html

```diff
@@ -1,11 +1,19 @@
 <form class="form" [formGroup]="formSignIn" (ngSubmit)="onSubmit()">
   <div class="form-row">
     <label class="label text-capitalize">email</label>
-    <input type="text" class="input  input-block">
+    <input type="text" formControlName="email" class="input input-block">
+    <ng-container *ngIf="isInvalidField('email')">
+      <wlx-message-error [control]="getControl('email')" [messages]="messages.email">
+      </wlx-message-error>
+    </ng-container>
   </div>
   <div class="form-row">
     <label class="label text-capitalize">password</label>
-    <input type="password" class="input  input-block">
+    <input type="password" formControlName="password" class="input input-block">
+    <ng-container *ngIf="isInvalidField('password')">
+      <wlx-message-error [control]="getControl('password')" [messages]="messages.password">
+      </wlx-message-error>
+    </ng-container>
```

Comment:
wlx-message-error already has a ng-container tag. ¿Is another one necessary? I'm speaking about the outer ng-container.